### DISCUSSION TITLE
Hovercard not working when focused from keyboard

### DISCUSSION THREAD

**Discussion by mule-stand** [ariakit/ariakit#3470] [Questions] [answered]
Why, when focused from keyboard, hovercard doesnt open?

**Comment by georgekaran** [comment, marked as answer]
Hello @mule-stand,

The `Hovercard` component is designed to work exclusively with mouse input, which means it won't respond to keyboard navigation. However, we do have an example that shows how you can display the `Hovercard` when the focus event is triggered. You can find this example at https://ariakit.org/examples/hovercard-disclosure.
